src/agent/web.ts

```diff
@@ -416,4 +417,34 @@ export default class Web extends WebAPI {
   async listAgentsForTransfer(chat_id: string): Promise<AgentForTransfer[]> {
     return this.send("list_agents_for_transfer", { chat_id });
   }
+
+  /**
+   * Marks a customer as followed. As a result, the requester (an agent) will receive
+   * the info about all the changes related to that customer via pushes.
+   * @param id - ID of customer
```

Comment:
opt:
```suggestion
   * @param id - ID of customer to follow
```
This change is to align with other param descriptions

Reply:
Fixed. Completed the description for parameters.